charts/internal/shoot-system-components/charts/csi-alicloud/templates/crd-volumesnapshots.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
   annotations:
     api-approved.kubernetes.io: "https://github.com/kubernetes-csi/external-snapshotter/pull/260"
     resources.gardener.cloud/delete-on-invalid-update: "true"
```

Comment:
Maybe this should be removed? 👀

Reply:
This annotation was introduced to make the update from K8s v1.16 to v1.17 (from external-snapshotter@v1 to external-snapshotter@v2) possible. If we remove it now, I guess this will regress such update. I now created https://github.com/gardener/gardener-extension-provider-alicloud/issues/314 with more detailed description. So I guess the safer option is to drop support for v1.15.x and v1.16.x and then remove this annotation from the volumesnapshot CRDs. WDYT?

Reply:
Sorry for the late reply. Sure, I'm fine. Thanks!